docs/community/com-config/com-config-cli-enable-sql-endpoint.md

```diff
@@ -1,35 +1,38 @@
 ---
-title: SQL Endpoint
+title: FeatureBase SQL CLI
 layout: default
 parent: Community configuration
 grand_parent: Community
 ---
 
-# SQL Endpoint
+# FeatureBase SQL Command Line Interface (CLI)
 
 The SQL endpoint allows you to execute SQL statements:
 * on the command-line
 * over an HTTP endpoint
 
 ## Before you begin
 
+* [Start the FeatureBase Community server](/docs/community/com-startup-connect) with the `--sql.endpoint-enabled` flag
 * CD to the `/featurebase/opt` directory
```

Comment:
Is this a thing?

Reply:
As far as being able to ingest with SQL, it's required. I also note this in the release notes on Github...